src/cpp/session/modules/SessionFiles.cpp

```diff
@@ -1397,8 +1409,9 @@ Error initialize()
    // subscribe to events
    events().onClientInit.connect(bind(onClientInit));
 
-   RS_REGISTER_CALL_METHOD(rs_readLines, 1);
-   RS_REGISTER_CALL_METHOD(rs_pathInfo, 1);
+   RS_REGISTER_CALL_METHOD(rs_getACP);
```

Comment:
Is there a reason for removing the 1 from the RS_REGISTER_CALL_METHOD calls?

Reply:
Those are sort of unneeded historical artifacts. Back when we were compiling against C++98, we didn't have a clean way of figuring out how many parameters each call method received, and so we have to explicitly declare it. That was a bit error-prone, as we occasionally forgot to update our registration code whenever we updated / extended an existing `rs_` method.

Now that we can figure it out using C++11 tooling (https://github.com/rstudio/rstudio/blob/89d90af57611a1d5e42b3dfdd2abf3565d029ea1/src/cpp/r/include/r/RRoutines.hpp#L80) I've been slowly cleaning up some of those usages as I see them.